test/integration/options/watch.spec.js

```diff
@@ -19,38 +21,155 @@ describe('--watch', function() {
       }
```

Comment:
Only the first test fails on Windows, the three remaining ones run successfully. So no need to skip these tests.

Reply:
Fixed in https://github.com/mochajs/mocha/pull/3915/commits/97de23f5f239e08101de766bf9c1fc9ee88fbe2a